spec/controllers/imports/issues_controller_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe Imports::IssuesController, type: :controller do
+  let(:patient) { create(:patient) }
+
+  describe "PatientLoggingConcern" do
```

Comment:
It feels like this is going to be similar for all of these controllers, could we create [a shared example](https://rspec.info/features/3-13/rspec-core/example-groups/shared-examples/) that we could then use like:

```rb
it_behaves_like "a controller that logs the patient ID" do
  ... # any controller specific set up
end
```